src/components/MetadataEditor/MetadataEditor.js

```diff
@@ -43,16 +54,15 @@ class MetadataEditor extends Component {
       label,
       summary,
       colour,
-      startTime,
-      endTime,
+      startTime: moment(startTime)
+        .utc()
+        .format(DISPLAY_TIME_FORMAT),
+      endTime: moment(endTime)
+        .utc()
+        .format(DISPLAY_TIME_FORMAT),
```

Comment:
How does moment deal with different time-zones. Will it try and add time-zone offsets to times written in UTC depending on where your browser is.

Reply:
irrelevant, just removed it because of the previous request: https://github.com/digirati-co-uk/timeliner/pull/105#discussion_r221549477 used `date-fns/format` instead

Reply:
moment(endTime).utc() === formatDate(new Date(endtime + (new Date().getTimezoneOffset() * 60 * 1000)))